secretsstoresynccontroller/config/samples/SecretStoreSecret2.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: secretsync.secret-sync.x-k8s.io/v1alpha1
+kind: SecretSync
+metadata:
+  name: <secret2-name-in-cluster>                     # name of the secret in the cluster
+spec:
+  secretProviderClassName: spc-multiple-secrets       # [REQUIRED] name of the SecretProviderClass
+  serviceAccountName: <workload-identity-sa-name>     # [REQUIRED] name of the service account
```

Comment:
it is used when the same SPC needs to be mounted in 2 different pods that have different SA. I don't think it's done explicitly but because the pods already have a SA configured they would just use that as-is.